rsdp/src/handler.rs

```diff
@@ -20,14 +20,14 @@ where
     H: AcpiHandler,
 {
     /// Construct a new `PhysicalMapping`.
-    /// `mapped_length` may differ from `region_length` if padding is added for alignment.
+    /// `mapped_length` describes usable span from `virtual_start` to end of last mapped page (for paging), and thus may exceed `physical_length`.
```

Comment:
A newline should precede this if you meant it to be on the line in the documentation after 'Construct a new `PhysicalMapping`.'
```suggestion

    /// `mapped_length` describes usable span from `virtual_start` to end of last mapped page (for paging), and thus may exceed `region_length`.
```